coordinatorlib/src/main/resources/com.socrata.datacoordinator.truth.schema/20220415-add-column-rollup-raw-soql.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+    <changeSet author="Chi" id="20220415-add-column-rollup-raw-soql">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="rollup_map" columnName="raw_soql"/>
+            </not>
+        </preConditions>
+        <addColumn tableName="rollup_map">
+            <column name="raw_soql" type="varchar"/>
```

Comment:
Vaguely surprised you're using `varchar` here. Or does that no longer have a reachable max length?

Reply:
This field mirrors rollup_map.soql which is varchar.